android/libraries/rib-base/src/androidTest/java/com/badoo/ribs/test/util/Instrumentation.kt

```diff
@@ -0,0 +1,5 @@
+package com.badoo.ribs.test.util
+
+import android.support.test.InstrumentationRegistry
+
+fun runOnMainSync(block: () -> Unit) = InstrumentationRegistry.getInstrumentation().runOnMainSync(block)
```

Comment:
I think this file can be merged with the `AndroidTestUtils` one or the other one can be renamed to `Activity.kt`